app/src/main/java/com/example/musicappdemo/notification/NotificationUtils.kt

```diff
@@ -0,0 +1,76 @@
+package com.example.musicappdemo.notification
+
+import android.annotation.SuppressLint
+import android.app.Notification
+import android.app.PendingIntent
+import android.content.Context
+import android.content.Intent
+import android.widget.RemoteViews
+import androidx.core.app.NotificationCompat
+import com.example.musicappdemo.ui.play.MainActivity
+import com.example.musicappdemo.R
+import com.example.musicappdemo.data.model.Song
+import com.example.musicappdemo.service.SongService
+
+@SuppressLint("RemoteViewLayout")
+fun senNotification(context: Context, song: Song, isPlayMusic: Boolean): Notification {
+    var intent: Intent = Intent(context, MainActivity::class.java)
+    var pendingIntent: PendingIntent = PendingIntent.getActivity(
+        context,
+        SongService.REQUEST_CODE, intent, PendingIntent.FLAG_UPDATE_CURRENT
```

Comment:
```suggestion
    var pendingIntent = PendingIntent.getActivity(
        context,
        SongService.REQUEST_CODE, intent, PendingIntent.FLAG_UPDATE_CURRENT
```

sua lai het may cho khai bao nha